Documentation/ExtensionInstallation/Index.rst

```diff
@@ -24,15 +24,14 @@ installation without Composer <extension-installation_without_composer>`.
 Extension installation in Composer mode
 =======================================
 
-Starting with TYPO3 11.4 if composer is used, all extensions, including the
-site package extension must be installed via Composer.
+From TYPO3 11.4 if composer is used, all extensions, including our
```

Comment:
```suggestion
From TYPO3 v11.4 if composer is used, all extensions, including our
```

I had to read the sentence twice. Maybe we can rephrase it, like "In Composer-based installations, all extensions ...".